201-specialized-vm-in-existing-vnet/azuredeploy.json

```diff
@@ -0,0 +1,149 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion": "1.0.0.0",
+   "parameters": {
+     "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the VM"
+      }
+    },
+    "osType": {
+      "type": "string",
+      "allowedValues": [
+        "Windows",
+        "Linux"
+      ],
+      "metadata": {
+        "description": "Type of OS on the existing vhd"
+      }
+    },
+     "osDiskVhdUri": {
+     "type": "string",
+        "metadata": {
+          "description": "Uri of the existing VHD in ARM standard or premium storage"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "metadata": {
+        "description": "Size of the VM"
+      }
+    },
+    "virtualNetworkName": {
```

Comment:
rename this to existingVirtualNetworkName to set the right expectations and remove any ambiguity